docs/user/azure/install_upi_azurestack.md

```diff
@@ -229,6 +229,16 @@ stringData:
   azure_region: <$REGION>
 ```
 
+NOTE: Any credentials for a credential request in Tech Preview must be excluded, or they will cause the
```

Comment:
Should we consider moving this Note/Comment up or highlighting it differently as it is inserted after a large file block which means it can be skipped quite easily?

Reply:
I agree this could be an improvement but I don’t think we need to fine tune the Azure Stack docs too much. I think we should just merge this as is.

Reply:
I am OK to merge it as is since it is in the Cluster Credentials section where an user might look in case of failures due to cluster credentials.